lib/app/utils/terminology.rb

```diff
@@ -145,6 +152,75 @@ def self.load_validators(directory = 'resources/terminology/validators/bloom')
       end
     end
 
+    # Parse the expansions that are in FHIR Models into valueset validators
+    def self.load_fhir_models_expansions
+      Inferno.logger.debug 'Loading FHIR Models Expansions'
+      FHIR::Definitions.expansions.each do |expansion|
+        url = expansion['url']
+        next if @known_valuesets[url]
+        next if SKIP_SYS.include? url
+
+        valueset = Inferno::Terminology::Valueset.new(@db)
+        valueset.valueset_model = FHIR::ValueSet.new(expansion)
+        valueset.vsa = self
+
+        @known_valuesets[valueset.url] = valueset
```

Comment:
nothing below this line is needed.  The `create_validators` call in the rake task will take care of it.